File: 240508/학점 계산기/credit-calculator.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
//

        double arr[] = new double[10];
//        String str[] = new String[10];
        
        int n = sc.nextInt();

        double sum = 0;
        double avg = 0;
        
        for (int i = 0; i < n; i++) {
            arr[i] = sc.nextDouble();
            sum+= arr[i];
        }
        
        
        
        avg = sum / n;

                System.out.printf("%.1f\n",avg);

        
        if(avg >= 4.0){
            System.out.println("Perfect");
        }else if(avg >= 3.0){
            System.out.printf("Good");
        }else{
            System.out.println("Poor");
        }
        
    }
}

File: 250219/학점 계산기/credit-calculator.py
n = float(input());

arr = list(map(float, input().split()));

avg = sum(arr)/n;

score = "";

if avg >= 4.0 :
    score = "Perfect";
elif avg >= 3.0 :
    score = "Good";
else :
    score = "Poor";

print(f"{avg:.1f}\n{score}");


